Resources/Prototypes/Entities/Mobs/Species/human.yml

```
@@ -26,107 +26,1214 @@
+      back-body-slim-f-new:
+        sizeMaps:
+          32:
+            sprite: _Sunrise/Mobs/Species/Human/New/Slim/displacement.rsi
+            state: back
+      head-body-slim-f-new:
+        sizeMaps:
+          32:
+            sprite: _Sunrise/Mobs/Species/Human/New/Slim/displacement.rsi
+            state: head
+
+# Old
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)